spec/lib/tasks/remove_spec.rb

```diff
@@ -12,13 +12,16 @@
     it "delegates to RemoveDocumentService" do
       note = "The reason the edition is being removed"
 
-      ClimateControl.modify NOTE: note do
-        Rake::Task["remove:gone"].invoke(edition.content_id)
-      end
-
-      expect(RemoveDocumentService).to have_received(:call) do |removed_edition, removal|
-        expect(removed_edition).to eq(edition)
-        expect(removal.explanatory_note).to eq(note)
+      travel_to(Time.zone.now) do
```

Comment:
you can achieve the same as this with `freeze_time do`

Reply:
Changed 👍